Library/Logger/Formatter/Stream.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Disturb\Logger\Formatter;
```

Comment:
I think you've not created your br from poc/head, the NS has been changed  to Vpg\Disturb